scripts/dappnode_install_pre.sh

```diff
@@ -164,6 +175,15 @@ else
     install_lsof 2>&1 | tee -a $LOG_FILE
 fi
 
+if [[ $SKIP_UNNATENDED_UPGRADES != "true"  ]]; then 
```

Comment:
when would be used this var for either true and false?

Reply:
Idea for that variable is to use it while starting script like:
```
SKIP_UNNATENDED_UPGRADES=true ./dappnode_install_pre.sh
```
After I fix typo, of course 😅 
Otherwise variable is empty and expression evaluates to false and package is installed.